ciao-cli/tenant.go

```diff
@@ -210,3 +388,61 @@ func listTenantResources(t *template.Template) error {
 
 	return nil
 }
+
+func listTenantConfig(t *template.Template, tenantID string) error {
+	if tenantID == "" {
+		fatalf("Missing required -for-tenant parameter")
```

Comment:
or --tenant-id when the user is not privileged.

Reply:
I'll update the error message, thanks.